ComponentKit/HostingView/CKComponentRootView.m

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "CKComponentRootView.h"
+#import "CKComponentRootViewInternal.h"
+
+#import "CKAssert.h"
+
+@implementation CKComponentRootView
+
+static NSMutableArray *hitTestHooks;
```

Comment:
Ohhh… I thought I saw it marked as an instance method, but it as a class method makes total sense.